pom.xml

```diff
@@ -43,6 +43,11 @@
     <url>https://github.com/rabbitmq/rabbitmq-java-client.git</url>
     <connection>scm:git:https://github.com/rabbitmq/rabbitmq-java-client.git</connection>
   </scm>
+  
+  <organization>
+    <name>Rabbit Technologies Ltd.</name>
```

Comment:
Rabbit Technologies has been acquired by SpringSource/VMware in 2010. Please use `Pivotal Software, Inc`.